pages/events/index.vue

```diff
@@ -157,10 +183,39 @@ export default class extends QiskitPage {
     img: '/images/events/no-events.svg'
   }
 
+  // multiselect
+  regionOptions = this.getOptions(this.regions)
+  typeOptions = this.getOptions(this.types)
+  theme: string = 'light'
+  labelRegions: string = 'All locations'
+  labelTypes: string = 'All types'
+  inline: boolean = false
+  feedback: string = 'fixed'
+
   get noEvents (): boolean {
     return (this as any).filteredEvents.length === 0
   }
 
+  getOptions (optionsList) {
+    return optionsList.map((item) => {
+      return {
+        label: item,
+        value: item,
+        name: item
+      }
+    })
+  }
+
+  updateWholeFilter (filter: string, filterValues: object) {
```

Comment:
Be more specific and use `Array` instead of `object` and add the return type, please